app/code/Magento/Bundle/Ui/DataProvider/Product/BundleDataProvider.php

```diff
@@ -24,17 +32,19 @@ class BundleDataProvider extends ProductDataProvider
      * @param string $requestFieldName
      * @param CollectionFactory $collectionFactory
      * @param Data $dataHelper
-     * @param \Magento\Ui\DataProvider\AddFieldToCollectionInterface[] $addFieldStrategies
-     * @param \Magento\Ui\DataProvider\AddFilterToCollectionInterface[] $addFilterStrategies
+     * @param StockStateInterface $stockState
      * @param array $meta
      * @param array $data
+     * @param \Magento\Ui\DataProvider\AddFieldToCollectionInterface[] $addFieldStrategies
+     * @param \Magento\Ui\DataProvider\AddFilterToCollectionInterface[] $addFilterStrategies
```

Comment:
Please remove the duplicate parameter declaration

Reply:
Sorry, that is not a duplicated declaration. They are different.

Reply:
Sorry, my bad, you are absolutely right!